deployments/kubernetes/chart/proxyinjector/templates/configmap.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    app: {{ template "proxyinjector-name" . }}
+    version: {{ .Chart.Version }}
+    group: {{ .Values.proxyinjector.labels.group }}
+    provider: {{ .Values.proxyinjector.labels.provider }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: {{ .Release.Name | quote }}
+    heritage: {{ .Release.Service | quote }}
+  name: {{ template "proxyinjector-name" . }}
+data:
+  {{- with .Values.keycloakProxyConfigmap.config }}
+  config.yml: |-
+    client-id: {{ .clientId }}
```

Comment:
We can just use toYaml here to pick up the whole config file from values.yaml
That way its up to the user to properly add config values and we won't have to manually add a new property whenever it comes in